package-lock.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
guh hop on pnpm (update the lockfile)

Reply:
Guh i used pnpm and lockfile remained the same for whatever reason

Reply:
it wont be in lockfile cause hermesc is peer dependency